### PR TITLE
Changed queries to always return ids instead of patient_ids

### PR DESCRIPTION
**Pull request by Vegemash**

### CODE REVIEW

File: protected/modules/OphCiExamination/models/PatientDiagnosisParameter.php [hunk 1]
```diff
@@ -129,8 +129,8 @@ public function renderParameter($id)
      */
     public function getIds()
     {
-        $queryStr = "
-            SELECT episode.patient_id
+        $queryStr = /**@lang nothing*/"
```
Comment (marshl) [inline protected/modules/OphCiExamination/models/PatientDiagnosisParameter.php]: This seems... overzealous.

File: protected/modules/OphCiExamination/models/PatientDiagnosisParameter.php [hunk 2]
```diff
@@ -129,8 +129,8 @@ public function renderParameter($id)
...
+            SELECT episode.patient_id as id
```
Comment (marshl) [inline protected/modules/OphCiExamination/models/PatientDiagnosisParameter.php]: "as" should be all caps

**Review by marshl — APPROVED** [review]

**Review by marshl — CHANGES_REQUESTED** [review]
We might need to fix those failing unit tests

**Comment by Vegemash** [comment]
the failing unit tests are addressed in the other PR

**Comment by marshl** [comment]
The unit tests are still failing. Would you have to rebase this commit on top of the target branch? I'm not sure if that will help.

**Comment by apigram** [comment]
@marshl By the looks of it, the OEA-450 PR is where the unit tests are fixed.

**Comment by apigram** [comment]
Is this review going to be abandoned, @Vegemash?

**Comment by Vegemash** [comment]
@apigram I'll just do the work again

**Pull request closed without merging**